python/dgl/graphbolt/feature_store.py

```diff
@@ -85,18 +87,35 @@ def __init__(self, feature_dict: dict):
         ... torch.ones(3, dtype=torch.long), torch.tensor([0, 1, 2]))
         >>> feature_store.read("user", torch.tensor([0, 1, 2]))
         tensor([1, 1, 1])
+
+        >>> import numpy as np
+        >>> user = np.arange(0, 5)
+        >>> item = np.arange(0, 6)
+        >>> np.save("/tmp/user.npy", user)
+        >>. np.save("/tmp/item.npy", item)
+        >>> feature_dict = {
+        ...     "user": torch.as_tensor(np.load("/tmp/user.npy",
```

Comment:
How does the np.load work, does it load all file to memory or it does some smart dispatching?

Reply:
If we specify the mmap_mode, it keeps the file on disk and reads the data when necessary. See the mode description in `memmap` (https://numpy.org/doc/stable/reference/generated/numpy.memmap.html#numpy.memmap).

Reply:
I see, thanks. Then we just need to make sure we understand the performance since this will be our recommended canonical solution.